### ISSUE TITLE
Add city wall lines

### ISSUE THREAD

**Issue by nvkelso** [mapzen/vector-datasource#188]
Old cities have old walls, let's show them on the map:

Background reading:

https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dcity_wall

**Comment by nvkelso** [comment]
The `boundaries` layer seems to make the most sense for this.

**Comment by zerebubuth** [comment]
After #329, and with the following scene file snippet:

```yaml
        city_walls:
          filter: { kind: city_wall }
          draw:
            lines:
              interactive: true
              order: 8
              color: '#ff0000'
              width: 4px
```

Around [George's Island](https://www.openstreetmap.org/#map=18/44.64117/-63.55951), I get:

## Before

![georges_island_z18_before](https://cloud.githubusercontent.com/assets/271360/10703299/f86d8970-79c6-11e5-912f-2ef0ca6eb348.jpg)

## After

![georges_island_z18_after](https://cloud.githubusercontent.com/assets/271360/10703303/fe3d99bc-79c6-11e5-89df-664f234ea999.jpg)

There are some other features visible in the OSM carto map, which are tagged as `barrier=retaining_wall`. Let me know if you want them included too.

**Comment by nvkelso** [comment]
Might as well pick those up at the same time, but seems like their default zoom should be later than city walls?

**Comment by nvkelso** [comment]
Retaining walls and paths seem like similar level of detail. We show Paths starting at zoom 15, let's match that.

**Comment by nvkelso** [comment]
@zerebubuth what zoom do city walls start coming in? Zoom 10+ seems like a good starting point.

**Comment by zerebubuth** [comment]
City walls start coming in at z14, which matches OSM Carto. Having them come in earlier is definitely possible, although z10 sounds a bit early to me. For comparison z10 is when we start to include county boundaries and secondary roads in the tiles. @nvkelso: What's the appropriate zoom to start bringing them in?

**Comment by zerebubuth** [comment]
Screenshot after 9fc86df in #329. The thinner red lines are the retaining walls.

![georges_island_z18_after_2](https://cloud.githubusercontent.com/assets/271360/10728821/1995d90a-7bdd-11e5-9778-ad1b85adfba0.jpg)

**Comment by nvkelso** [comment]
They should be available for styling by **zoom 12** (that's when we show minor roads, and these play into that hierarchy). We might only show them in style at zoom 14, but generally our practice is to include features a few zooms earlier than we need them.

**Comment by zerebubuth** [comment]
Cool. Fixed in 6eba697.
